app/Models/NetworkCampaign.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+namespace Adshares\Adserver\Models;
+
+use Adshares\Adserver\Models\NetworkBanner;
+use Adshares\Adserver\Models\NetworkCampaignExclude;
+use Adshares\Adserver\Models\NetworkCampaignRequire;
+
+use Adshares\Adserver\Models\Traits\AutomateMutators;
+use Adshares\Adserver\Models\Traits\BinHex;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\DB;
+
+class NetworkCampaign extends Model
+{
+    use AutomateMutators;
+    use BinHex;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+      'uuid',
+      'source_created_at', 'source_updated_at',
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "source_updated_at" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)